cmd/podman/images/buildx.go

```diff
@@ -0,0 +1,29 @@
+package images
+
+import (
+	"github.com/containers/podman/v3/cmd/podman/registry"
+	"github.com/containers/podman/v3/cmd/podman/validate"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Command: podman _buildx_
+	// This is a hidden command, which was added to make converting
+	// from Docekr to Podman easier.
```

Comment:
typo Docker

Reply:
Fixed

Reply:
@Luap99 aotocompletion tests are still not happy.